frontend/js/src/pins/PinRecordingModal.tsx

```diff
@@ -111,7 +118,44 @@ export default NiceModal.create(
       },
       [recordingToPin, blurbContent]
     );
-
+    const updatePinnedRecordingComment = React.useCallback(
+      async (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        try {
+          if (rowId && recordingToPin && currentUser?.auth_token) {
+            const response = await APIService.updatePinRecordingBlurbContent(
+              currentUser.auth_token,
+              rowId,
+              blurbContent
+            );
+            if (!response.status) {
+              toast.error(
+                <ToastMsg
+                  title="Comment update failed"
+                  message="Something went wrong"
+                />,
+                {
+                  toastId: "pin-update-failed",
+                }
+              );
+            } else {
+              toast.success(
+                <ToastMsg
+                  title="Comment updated"
+                  message={`Comment has been updated: ${blurbContent}`}
```

Comment:
This content could potentially be very long, so I don't think we want to add it into the toast here.
I think we could even drop the message here and let the title do all the work.